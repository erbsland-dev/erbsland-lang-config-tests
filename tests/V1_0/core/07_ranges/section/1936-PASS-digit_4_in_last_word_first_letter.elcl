# Erbsland Configuration Language Test File
[name_4name]