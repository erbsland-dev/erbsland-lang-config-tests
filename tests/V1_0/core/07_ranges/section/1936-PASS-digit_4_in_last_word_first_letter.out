name_4name = SectionWithNames()
